traffic-control-openshift/README.md

```diff
@@ -0,0 +1,62 @@
+# Traffic Control on OpenShift
+
+## What do you need to start?
+- Get a OpenShift ready VM [here](https://github.com/kinvolk/openshift-evangelists-vagrant-origin) to run the demo. You need to __reboot__ the VM after you build it.
+- Get get the demo repository [here](https://github.com/kinvolk/demo)
+- Copy the .yaml files into the VM
```

Comment:
give the command line how to copy the .yaml files because below (line 54), you expect them to be in the `apps/` directory.

Also, you could move that below the ssh-config section. Then users can use "scp" command directly.